.gitignore

```diff
@@ -1,6 +1,8 @@
 ## Ignore Visual Studio temporary files, build results, and 
 ## files generated by popular Visual Studio add-ons.
 
+.idea
```

Comment:
Care to explain why it's needed?

Reply:
Not required but I'm using Rider as my IDE and wasn't wanting to commit the files it added. I thought adding it here would help any others using Rider too